samples/cpp/benchmark_app/main.cpp

```diff
@@ -205,6 +205,21 @@ int main(int argc, char* argv[]) {
             slog::info << "GPU extensions is loaded " << ext << slog::endl;
         }
 
+        if (FLAGS_hint.empty()) {
```

Comment:
nstreams is never ignored, so I meant "setThroughputStreams should be cleaned from the code that sets the STREAMS_AUTO"- so that code-path doesn't interfere with the hints

Reply:
Just update previous comments for clarification.

If device support PERFORMANCE_HINTS, benchmark app will set throughput hints as default configuration instead of other internal setting in app.

This change will not impact any configuration which application set through SetConfig() or LoadNetwork().